Screens/ParkedCarForm.jsx

```diff
@@ -7,47 +7,104 @@ import {
   View,
 } from "react-native";
 import { Picker } from "@react-native-picker/picker";
-import { useContext, useState } from "react";
+import { useContext, useState, useCallback } from "react";
 import UploadImage from "../Components/UploadImage";
 import { getCurrentDateTime } from "../utils/utils";
 import { ParkingFormModal } from "../Components/Modals/ParkingFormModal";
+import { FIRESTORE_DB } from "../firebaseConfig";
+import { Timestamp, collection, doc, addDoc, setDoc, query, orderBy, limit, getDocs } from "firebase/firestore";
+
 
 const ParkedCarForm = () => {
   const [duration, setDuration] = useState("60");
   const [reminder, setReminder] = useState("5");
   const [notes, setNotes] = useState("");
 
+  const handleSubmit = useCallback(async () => {
+    try {
+      const parkedCarsCollection = collection(FIRESTORE_DB, "parkings");
+
+      const parkDoc = {
+        duration,
+        reminder,
+        notes,
+        timestamp: Timestamp.now(),
+      };
+
+      const { id } = await addDoc(parkedCarsCollection, parkDoc);
+
+
+
+      console.log("Data submitted successfully!");
+    } catch (error) {
+      console.error("Error submitting data:", error);
+    }
+  }, [duration, reminder, notes]);
+
   return (
-    <KeyboardAvoidingView>
-      <Text>ParkedCarForm</Text>
+    <KeyboardAvoidingView style={styles.container}>
+      <Text style={styles.title}>ParkedCarForm</Text>
       <UploadImage />
-      <Text>Form incoming {getCurrentDateTime()}</Text>
-      <Text>I'm going to park</Text>
+      <Text style={styles.subtitle}>Form incoming {getCurrentDateTime()}</Text>
```

Comment:
Why is this still here? Isn't this a placeholder for the form?

Reply:
Please elaborate ? current date and time is going to be stored in the db and shown on the form ? It's not styled yet as we are going for functionality first :)